org.hl7.fhir.validation/src/test/java/org/hl7/fhir/validation/tests/ValidationTests.java

```diff
@@ -524,11 +531,37 @@ public void test() throws Exception {
           Assert.assertTrue(fp.evaluateToBoolean(null, le, le, le, fp.parse(exp)));
         }
       }
-      checkOutcomes(errorsLogical, logical, "logical", name, suppress);
+      checkOutcomes(errorsLogical, logical, "logical", "logical", name, suppress);
     }
     logger.verifyHasNoRequests();
   }
 
+  private void moveOutcomes(JsonObject focus, String name, String mode) throws IOException {
+    if (REVISING_TEST_CASES) {
+      if (focus.has("dotnet-brianpos") && focus.get("dotnet-brianpos").isJsonObject()) {
+        byte[] json = JsonTrackingParser.writeBytes(focus.getAsJsonObject("dotnet-brianpos").getAsJsonObject("outcome"), true);
+        String filename = Utilities.path("/Users/grahamegrieve/work/test-cases/validator/outcomes/brianpos", name.replace("/", "-")+"-"+mode+".json"); 

```

Comment:
Should this have been committed to the repo?

Reply:
it doesn't belong long term, but there's a collection of tests I need to make decisions about and then re-run that code. and I didn't want to lose it and have to rewrite it